quartz/i18n/locales/vi-VN.ts

```diff
@@ -3,85 +3,83 @@ import { Translation } from "./definition"
 export default {
   propertyDefaults: {
     title: "Không có tiêu đề",
-    description: "Không có mô tả được cung cấp",
+    description: "Không có mô tả",
   },
   components: {
     callout: {
-      note: "Ghi Chú",
-      abstract: "Tóm Tắt",
+      note: "Ghi chú",
+      abstract: "Tổng quan",
       info: "Thông tin",
-      todo: "Cần Làm",
-      tip: "Gợi Ý",
-      success: "Thành Công",
-      question: "Nghi Vấn",
-      warning: "Cảnh Báo",
-      failure: "Thất Bại",
-      danger: "Nguy Hiểm",
+      todo: "Cần phải làm",
+      tip: "Mẹo",
+      success: "Thành công",
+      question: "Câu hỏi",
+      warning: "Cảnh báo",
+      failure: "Thất bại",
+      danger: "Nguy hiểm",
       bug: "Lỗi",
-      example: "Ví Dụ",
-      quote: "Trích Dẫn",
+      example: "Ví dụ",
+      quote: "Trích dẫn",
     },
     backlinks: {
-      title: "Liên Kết Ngược",
-      noBacklinksFound: "Không có liên kết ngược được tìm thấy",
+      title: "Liên kết ngược",
+      noBacklinksFound: "Không có liên kết ngược nào",
     },
     themeToggle: {
-      lightMode: "Sáng",
-      darkMode: "Tối",
+      lightMode: "Chế độ sáng",
+      darkMode: "Chế độ tối",
     },
     readerMode: {
       title: "Chế độ đọc",
     },
     explorer: {
-      title: "Trong bài này",
+      title: "Nội dung",
```

Comment:
Not necessary Nội dung, because this is just the title for explorer, so the original makes more sense contextually.

Reply:
This is not used for site navigation, this is actually just the "folder view" of the site rather.